src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,14 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String getPrintableString(Printable...printables) {
```

Comment:
you can add a header comment to this method as the behavior is different as compared to the one inside the `Printable` interface